src/main/resources/assets/tconstruct/book/materials_and_you/en_us/tools/kama.json

```diff
@@ -4,12 +4,6 @@
     {
       "text": "The Kama is a precision reaping tool, mowing down plants and shearing animals."
     },
-    {
-      "text": "\n"
-    },
-    {
-      "text": "Rightclicking on crops or sugarcane harvests and replants all fully grown plants in the area."
-    }
   ],
   "properties": [
     "Harvests crops on right click",
```

Comment:
I feel we should avoid saying "right click" as controls may be remapped. Perhaps "Use on crops to harvest them" or "Interact with crops to harvest"? Sugar cane and kelp may still be worth a metion.

Should also mention that kamas can hoe dirt.